### ISSUE TITLE
stacking and time mode default breaks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josejaner/flot#333]
```
Flot breaks after adding the stack plugin to the TimeSeries example in 
flot-0.6 (line 333).

This appears to be an issue where the series.stack property is not set 
with any sort of default, but is referenced.  Defining: "series: 
{stack:false}" fixes the issue.


```

Original issue reported on code.google.com by `jeremy.r...@gmail.com` on 15 Apr 2010 at 3:13

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the late reply - it seems to work in the development version, so I'm 
going to assume it's fixed since the 0.6 release. Thanks for the report anyway.
```

Original comment by `olau%iol...@gtempaccount.com` on 13 Dec 2010 at 3:31
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dnsch...@gmail.com` on 4 Jun 2012 at 2:52
